packages/roosterjs-content-model/lib/editor/ContentModelEditor.ts

```diff
@@ -69,4 +71,24 @@ export default class ContentModelEditor
 
         return core.defaultFormat;
     }
+
+    /**
+     * Paste into editor using a clipboardData object
+     * @param clipboardData Clipboard data retrieved from clipboard
+     * @param pasteAsText Force pasting as plain text. Default value is false
+     * @param applyCurrentStyle True if apply format of current selection to the pasted content,
+     * false to keep original format.  Default value is false. When pasteAsText is true, this parameter is ignored
+     */
+    public paste(
```

Comment:
Is this necessary?

Reply:
No, we can remove it. It is only for testing in demo site. Since PasteOption Plugin from RoosterReact calls this.editor.paste. Will Remove it.

Reply:
For long term, we should have a separate demo site for content model.